frontend/src/metabase/admin/databases/components/DatabaseEditForms.jsx

```diff
@@ -1,27 +1,25 @@
 import React, { Component, PropTypes } from "react";
-
+import cx from "classnames";
 import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper.jsx";
 import DatabaseDetailsForm from "metabase/components/DatabaseDetailsForm.jsx";
 
-import cx from "classnames";
 
 export default class DatabaseEditForms extends Component {
-    constructor(props, context) {
-        super(props, context);
-        this.state = {
-            formSuccess: null,
-            formError: null
-        };
-    }
 
     static propTypes = {
         database: PropTypes.object,
         details: PropTypes.object,
         engines: PropTypes.object.isRequired,
         hiddenFields: PropTypes.object,
-        save: PropTypes.func.isRequired
+        selectEngine: PropTypes.func.isRequired,
+        save: PropTypes.func.isRequired,
+        formState: PropTypes.object
+    };
+
+    static defaultProps = {
+        formSuccess: null,
+        formError: null
     };
-    static defaultProps = {};
 
     async detailsCaptured(database) {
         this.setState({ formError: null, formSuccess: null })
```

Comment:
Yeah, I must have missed that.  I'll remove it since all that happens in the redux action now.